scripts/compile-tsc.js

```diff
@@ -1,12 +1,20 @@
 /* eslint-disable no-console */
+// tslint:disable:no-console
```

Comment:
still needed?

Reply:
There are log outputs in the compile scripts that are marked red without

Reply:
next PR: replace tslint with eslint everywhere, then it can be removed